### PR TITLE
 feat: dont display multiple attendee details recycler for single ticket

### LINKED ISSUE #501: dont show multiple attendee details section for single attendee
working on it 👍

### PR DESCRIPTION
**Pull request by simarsingh24**
fixes #501

### CODE REVIEW

**Comment by simarsingh24** [comment]
screenshot_20180808-013007_eventyay attendee

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 1]
```diff
@@ -64,6 +68,9 @@ class AttendeeFragment : Fragment() {
     private lateinit var cardBrand: String
     private var id: Long = -1
     private lateinit var API_KEY: String
+    var singleTicket = false
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: Can be private. specify the type as well

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt [hunk 2]
```diff
@@ -42,9 +48,11 @@ class AttendeeViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         itemView.attendeeItemLastName.addTextChangedListener(textWatcher)
         itemView.attendeeItemEmail.addTextChangedListener(textWatcher)
         itemView.attendeeItemCountry.addTextChangedListener(textWatcher)
+        fillInformationSection(attendeeRecyclerAdapter.customForm)
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt]: add blank line to make the code more readable

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt (hunk 1 above)
Comment (nikit19) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: @simarsingh24 Boolean

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt [hunk 3]
```diff
@@ -11,21 +11,28 @@ import org.fossasia.openevent.general.attendees.forms.CustomForm
...
 
-    fun bind(attendeeRecyclerAdapter: AttendeeRecyclerAdapter, forms: List<CustomForm>, formVisibility: Boolean, position: Int) {
         setText(itemView.attendeeItemCountry, attendeeRecyclerAdapter.attendeeList[position].country)
         setText(itemView.attendeeItemLastName, attendeeRecyclerAdapter.attendeeList[position].lastname)
         setText(itemView.attendeeItemEmail, attendeeRecyclerAdapter.attendeeList[position].email)
         setText(itemView.attendeeItemFirstName, attendeeRecyclerAdapter.attendeeList[position].firstname)
-        itemView.attendeeItemTicketName.text = "Ticket Name - " + attendeeRecyclerAdapter.ticketList[position].name
-        fillInformationSection(forms, formVisibility)
-        val textWatcher = object : TextWatcher {
+        itemView.attendeeItemTicketName.text = """Ticket Name - ${attendeeRecyclerAdapter.ticketList[position].name}"""
```
Comment (dreadpool2) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt]: change `"""` to `""`

Comment (simarsingh24) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt]: thats needed

**Pull request merged**